rich/pretty.py

```diff
@@ -524,6 +524,28 @@ def __str__(self) -> str:
             )
 
 
+def _is_namedtuple(obj: Any) -> bool:
```

Comment:
I wonder if we can be more precise than this. I know there is no `is_namedtuple` function, alas.